tap_csv/client.py

```diff
@@ -81,8 +81,16 @@ def is_valid_filename(self, file_path: str) -> bool:
     def get_rows(self, file_path: str) -> Iterable[list]:
         """Return a generator of the rows in a particular CSV file."""
         encoding = self.file_config.get("encoding", None)
+        csv.register_dialect("tap_dialect", 
```

Comment:
@Bartman0 this is great!

As I'm reviewing I'm realizing that its probably better to set defaults at the top level in [config definition](https://github.com/MeltanoLabs/tap-csv/blob/eaf47eff21a5cea407423ba2d25daeb3b31a4976/tap_csv/tap.py#L24) so the `--about` function can return them and default config values are set and organized consistently in one place. I created a follow on issue https://github.com/MeltanoLabs/tap-csv/issues/71 because its not something you have to change in this PR unless you want to.